core/src/main/scala/dagr/core/execsystem/TaskManager.scala

```diff
@@ -523,7 +528,29 @@ class TaskManager(taskManagerResources: TaskManagerResources = TaskManagerDefaul
     )
   }
 
+  /** Refreshes the top-like interface */
+  private class TerminalThread(terminal: TopLikeStatusReporter) extends Runnable {
```

Comment:
Best not to name things `Thread` when they implement `Runnable`, as the two are highly related, but not the same thing.